### DISCUSSION TITLE
Audio codecs

### DISCUSSION THREAD

**Discussion by rahulv07** [bluefireteam/audioplayers#1073] [Q&A] [answered]
What are the audio codecs that audioplayers can play? I played .wav codec using audioplayers. I need to know what are the other codecs, that audioplayers support?

Can it play the following codecs?
- SBC (low complexity subband code)
- mp3
- apt
- aptX

**Comment by erickzanardo** [comment, marked as answer]
That is defined by the native platforms, on our readme we have some links that will lead you to pages that list on android and ios some of the supported formats, but be aware that the codecs provided on android devices varies depending on the manufacturer, so my advice is to stick to the common file formats
